vault/changelog.md

```diff
@@ -6,6 +6,24 @@ updated: 1653971733422
 created: 1601508213606
 nav_order: 2
 ---
+
+## 0.99.0
+
+### Features
+- feat(structure): add set task status commands (#3033) @kaan ([[docs|dendron://dendron.dendron-site/dendron.ref.commands#set-task-status]])
```

Comment:
`commands` should be singular (`command`)

Reply:
ack